bot/exts/code_jams/_cog.py

```diff
@@ -222,6 +222,19 @@ async def unpin(self, ctx: commands.Context, message: discord.Message | None = N
         """Lets Code Jam Participants to unpin messages in their team channels."""
         await pin_flow(ctx, PIN_ALLOWED_ROLES, self.bot.code_jam_mgmt_api, message, True)
 
+    @codejam.command("ping")
+    @commands.has_any_role(Roles.admins, Roles.events_lead, Roles.code_jam_event_team, Roles.code_jam_participants)
+    @in_code_jam_category(_creation_utils.CATEGORY_NAME)
+    async def ping_codejam_team(self, ctx: commands.Context) -> None:
+        """Ping the team role for the channel this command is ran in."""
+        team_resp = await self.bot.code_jam_mgmt_api.get("/teams/find", params={"name": ctx.channel.name})
```

Comment:
The `/teams/find?name=<team name>` API endpoint doesn't handle dashes. If this command is invoked in a team channel with a multi-word team, eg. `cool-kids`, [the CJ MGMT system will run a (case insensitive) DB query for `cool-kids`](https://github.com/python-discord/code-jam-management/blob/5011a22166f413d2e006293f38c5af503ebf5aad/api/routers/teams.py#L49-L59). Problem is that the team is stored in the DB with the name `Cool Kids`. The casing doesn't matter, but the space/dash difference does and the find query fails with `404 Team Not Found`. 

![image](https://github.com/python-discord/sir-robin/assets/63936253/e8fa0e1a-9828-4427-89ed-da4a096828ba)
![image](https://github.com/python-discord/sir-robin/assets/63936253/9e3a28c8-b0a2-4382-b989-10ff3d6a03d9)

AFAICT there are three ways of fixing this:

1. Replace dashes with spaces before inserting it as a HTTP URL parameter
2. Do the same (or something slighter smarter?) but on the CJ MGMT System side
3. Store the team channel IDs when they're first created by Sir Robin in the DB and do a ID search

I'm not qualified to suggest which one is best.

Reply:
resolved in the forcepush. As you say, it would be nice to get a team via channel id and role id, since we store those too. One for next time I think 
https://github.com/python-discord/sir-robin/compare/e42b09c0a8dfbb6a397ecfbc9201e663b66b0bc5..0169ca74bd9e1151f2b5b122cdf67287131c01bf